lms/djangoapps/instructor_task/tasks.py

```diff
@@ -40,14 +40,15 @@
     cohort_students_and_upload,
     upload_course_survey_report,
     upload_ora2_data,
-    upload_proctored_exam_results_report
+    upload_proctored_exam_results_report,
+    upload_students_grades_data,
 )
 from lms.djangoapps.instructor_task.tasks_helper.module_state import (
     delete_problem_module_state,
-    perform_module_state_update,
```

Comment:
ditto, why the order is changed?

Reply:
Import order fixed by using `isort` lib.